src/components/DropdownMenu.tsx

```diff
@@ -0,0 +1,29 @@
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  onSelected: (person: Person) => void;
+};
+
+export const DropdownMenu: React.FC<Props> = ({ people, onSelected }) => {
+  const handleSelect = (person: Person) => {
+    onSelected(person);
+  };
+
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {people.map(person => (
+          <div
+            className="dropdown-item"
+            data-cy="suggestion-item"
+            key={person.name}
```

Comment:
Consider using a more unique identifier for the `key` prop, such as an `id` if available, instead of `person.name`. Using a non-unique key can lead to performance issues and unexpected behavior in React.